functionaltests/src/test/java/util/CustomWebDriver.java

```diff
@@ -14,7 +14,7 @@
     private ChromeDriver driver;
 
     CustomWebDriver() {
-        String currentDriver = SystemUtils.IS_OS_MAC ? "chromedriver_mac_79" : "chromedriver_linux_79";
+        String currentDriver = SystemUtils.IS_OS_MAC ? "chromedriver_mac_84" : "chromedriver_linux_84";
```

Comment:
shall we use the latest version of chromedriver? https://chromedriver.chromium.org/downloads

Reply:
d2fdef3f4

Reply:
9bf267347